Task 2/src/main/java/ru/cft/focusstart/kartashev/Parser.java

```diff
@@ -0,0 +1,48 @@
+package ru.cft.focusstart.kartashev;
+
+import java.io.*;
+
+class Parser {
+    private String inputFileName;
+    private Shape shape;
+
+    Parser(String inputFileName) {
+        this.inputFileName = inputFileName;
+    }
+
+    Shape getInputShape() {
+
+        try {
+            FileInputStream fstream = new FileInputStream(inputFileName);
+            BufferedReader br = new BufferedReader(new InputStreamReader(fstream));
```

Comment:
```suggestion
            try(BufferedReader br = new BufferedReader(new InputStreamReader(fstream))) {
```
У объекта fstream не вызывается метод close(); это чревато утечкой памяти. В таких случаях удобней использовать конструкцию try with resource